pkg/api/v1/types.go

```diff
@@ -684,6 +685,24 @@ type GCEPersistentDiskVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty" protobuf:"varint,4,opt,name=readOnly"`
 }
 
+// Represents a Quobyte mount that lasts the lifetime of a pod.
+// Quobyte volumes do not support ownership management or SELinux relabeling.
+type QuobyteVolumeSource struct {
+	// Registry represents a single or multiple Quobyte Registry services
+	// which acts as the central registry for volumes
+	Registry string `json:"registry" protobuf:"bytes,1,opt,name=registry"`
+
+	// Volume is the Quobyte volume name.
+	Volume string `json:"volume" protobuf:"bytes,2,opt,name=volume"`
+
+	// Path is the Quobyte volume path
```

Comment:
It's the path inside the Quobyte volume. If no path is specified the whole Quobyte Volume is mounted. If a path specified is only the directory inside the Quobyte Volume get's mounted.

E.q.:
Quobyte Volume has 3 paths:
/test
/test2
/test3

If I don't specify a path I can access all 3 directories and if I specify for e.q. `/test3` then the container can only access data inside `/test3`